include/crow/json.h

```diff
@@ -1243,9 +1268,35 @@ namespace crow
 #endif
 
         public:
-
             wvalue() : returnable("application/json") {}
 
+            wvalue(std::nullptr_t) : returnable("application/json"), t_(type::Null) {}
+
+            wvalue(bool value) : returnable("application/json"), t_(value ? type::True : type::False) {}
+
+            wvalue(std::uint8_t value) : returnable("application/json"), t_(type::Number), nt(num_type::Unsigned_integer), num(value) {}
```

Comment:
Maybe call `wvalue(static_cast<uint64_t>(value))` to reduce code amount? Similar for other types.